### PR TITLE
Add What's new content for 7.11 release

### PR DESCRIPTION
**Pull request by EamonnTP**
### Summary

This PR adds the What's new content for the 7.11 release.

The page includes content for the following features:

- APM service overview page
- APM and logs
- Enhanced host details view
- Synthetics waterfall view
- Runtime fields in logs
- Fleet and Elastic Agent (Beta 3)
- APM Prometheus metrics client
- Elastic Logging Plugin for Docker (GA)
- Filebeat and Metricbeat modules (GA)
- Natively collect AWS Fargate metrics
- Sync time across Observability apps
- Kibana alerting framework (GA)
- Searchable snapshots in Elastic Cloud (GA)
- Native support of CCR and CCS in Elastic Cloud

### HTML preview

https://observability-docs_366.docs-preview.app.elstc.co/guide/en/observability/7.11/whats-new.html

### Related issue

Closes https://github.com/elastic/observability-dev/issues/1298

### CODE REVIEW

File: docs/en/observability/whats-new.asciidoc [hunk 1]
```diff
@@ -3,19 +3,249 @@
...
 
-// What's new content goes in here.
+[discrete]
+== APM service overview page
+
+Understanding your services' performance and detecting a root cause just got easier
+with our new service overview page in the {apm-app}.
+
+In {minor-version}, the new service overview page is now your default starting point
+for investigating any services monitored by the Elastic APM agents or with OpenTelemetry and Jaeger.
```
Comment (bmorelli25) [inline docs/en/observability/whats-new.asciidoc:31]:
```suggestion
for investigating any services monitored by Elastic APM agents or with OpenTelemetry and Jaeger.
```

File: docs/en/observability/whats-new.asciidoc [hunk 2]
```diff
@@ -3,19 +3,249 @@
...
+
+[role="screenshot"]
+image::images/apm-service-overview.png[Service overview page]
+
+The improved APM service overview page provides you with: 
+
+* High-level details including service, runtime and framework versions, container,
+and Kubernetes information, along with cloud metadata, if applicable
```
Comment (bmorelli25) [inline docs/en/observability/whats-new.asciidoc:39]: I think "container and Kubernetes information" should be grouped together here

```suggestion
* High-level details including service, runtime and framework versions, container
and Kubernetes information, along with cloud metadata, if applicable
```

File: docs/en/observability/whats-new.asciidoc [hunk 3]
```diff
@@ -3,19 +3,249 @@
...
+* Latency timeline overlapped with deployment events, alert violations and anomalies
```
Comment (bmorelli25) [inline docs/en/observability/whats-new.asciidoc:40]:
```suggestion
* Latency timeline overlapped with deployment events, alert violations, and anomalies
```

File: docs/en/observability/whats-new.asciidoc [hunk 4]
```diff
@@ -3,19 +3,249 @@
...
+
+For {minor-version}, we have two exciting announcements:
+
+* Elastic Common Schema (ECS) logging libraries are now generally available! You can now use
+ECS logger plugins to format your logs into ECS-compatible JSON. Using these libraries, along
+with an Elastic APM agent, means that you do not have any manual configuration to correlate
+logs to your application traces.
+* Embedded trace logs are now available in the curated {apm-app}, which means you no longer have
+to navigate outside the trace view to visualize your application logs. These contextualized
+and correlated logs quickly point you to the underlying issues with your application.
```
Comment (bmorelli25) [inline docs/en/observability/whats-new.asciidoc:65]: ? Either works, I guess

```suggestion
and correlated logs quickly point you to the underlying issues within your application.
```

**Review by bmorelli25 — COMMENTED** [review]
A few small comments. Pushing this through now so that I can pull and try to figure out the build error.

**Comment by bmorelli25** [comment]
<details>
  <summary>Doc build error from an earlier build</summary>

```
08:09:28 INFO:build_docs:ERROR building Installation and Upgrade Guide version 7.11
08:09:28 INFO:build_docs:
08:09:28 INFO:build_docs:Recent commits in stack-docs/Installation and Upgrade Guide:7.11:docs/en
08:09:28 INFO:build_docs:------------------------------------------------------------------------
08:09:28 INFO:build_docs:0cdb56f2 - (7.11) [DOCS] Add scatterplot matrix to regression example (#1538) (#1547) (6 days ago) <Lisa Cawley>
08:09:28 INFO:build_docs:0f324e05 - [DOCS] Add known issue for zero memory on Debian 8 (#1528) (#1543) (6 days ago) <Lisa Cawley>
08:09:28 INFO:build_docs:f25972cb - [DOCS] Add scatterplot matrix to classification example (#1539) (#1542) (6 days ago) <Lisa Cawley>
08:09:28 INFO:build_docs:17288097 - [DOCS] Add scatterplot matrix to outlier detection example (#1507) (#1537) (11 days ago) <Lisa Cawley>
08:09:28 INFO:build_docs:5321b4d9 - Updates instructions on how to import models (#1535) (11 days ago) <Josh Devins>
08:09:28 INFO:build_docs:eca3e6e4 - [DOCS] Remove unused installation file (#1529) (#1533) (13 days ago) <Lisa Cawley>
08:09:28 INFO:build_docs:b9a7207d - [DOCS] Remove unnecessary license details (#1531) (13 days ago) <Lisa Cawley>
08:09:28 INFO:build_docs:dad4019e - [DOCS] Add Kibana breaking changes to Install Upgrade Guide (#1525) (3 weeks ago) <Lisa Cawley>
08:09:28 INFO:build_docs:a36fcbd7 - [DOCS] Add space limitations for ML objects (#1503) (#1524) (3 weeks ago) <Lisa Cawley>
08:09:28 INFO:build_docs:660e6d3d - [DOCS] Refresh screenshots in machine learning tutorial (#1506) (#1521) (3 weeks ago) <Lisa Cawley>
08:09:28 INFO:build_docs:
08:09:28 INFO:build_docs:Recent commits in apm-server/Installation and Upgrade Guide:7.11:docs/guide
08:09:28 INFO:build_docs:---------------------------------------------------------------------------
08:09:28 INFO:build_docs:478ab50e2 - docs: link to other troubleshooting guides (#4608) (#4614) (3 weeks ago) <Brandon Morelli>
08:09:28 INFO:build_docs:33e42d359 - docs: 7.11 breaking changes (#4600) (#4607) (3 weeks ago) <Brandon Morelli>
08:09:28 INFO:build_docs:e3ab6062b - docs: fix oTel link (#4578) (#4580) (6 weeks ago) <Brandon Morelli>
08:09:28 INFO:build_docs:f3f065860 - docs: Add sampling docs and improve distributed tracing (#4475) (#4566) (7 weeks ago) <Brandon Morelli>
08:09:28 INFO:build_docs:488450132 - [docs] Fix typo in Node.js addLabels method name (#4556) (#4564) (7 weeks ago) <Brandon Morelli>
08:09:28 INFO:build_docs:535cb419e - [docs] Add content for exporting OpenTelemetry metrics (#4498) (#4544) (7 weeks ago) <EamonnTP>
08:09:28 INFO:build_docs:6c47604c6 - docs: remove apm whats new (#4385) (#4386) (3 months ago) <Brandon Morelli>
08:09:28 INFO:build_docs:becac3b3d - Merge pull request #4288 from EamonnTP/update-logs-link (#4297) (4 months ago) <EamonnTP>
08:09:28 INFO:build_docs:22b304ca0 - docs: update release highlight include (#4253) (4 months ago) <Brandon Morelli>
08:09:28 INFO:build_docs:10fd5d946 - [7.x] docs: improve getting started experience (#4222) (#4251) (4 months ago) <Brandon Morelli>
08:09:28 INFO:build_docs:
08:09:28 INFO:build_docs:Recent commits in beats/Installation and Upgrade Guide:7.11:libbeat/docs
08:09:28 INFO:build_docs:------------------------------------------------------------------------
08:09:28 INFO:build_docs:3577861957 - [DOCS] Add setup content to Kubernetes and Cloud Foundry docs (#23580) (#23681) (6 days ago) <EamonnTP>
08:09:28 INFO:build_docs:0cc6ef377a - Update Golang to 1.14.14 (#23506) (12 days ago) <Noémi Ványi>
08:09:28 INFO:build_docs:a4bfc8dc51 - docs: Prepare Changelog for 7.10.2 (#23416) (#23519) (2 weeks ago) <Brandon Morelli>
08:09:28 INFO:build_docs:9ddcea44ac - [docs] Update misleading phrasing in filebeat's logging.metrics.enabled docs (#23373) (#23472) (3 weeks ago) <Brandon Morelli>
08:09:28 INFO:build_docs:6a1f74fcd8 - Cherry-pick to 7.11: DaemonSet is supposed to be camelCase (#23270) (#23433) (3 weeks ago) <DeDe Morton>
08:09:28 INFO:build_docs:007b47cfcc - [7.x] Sync changelog (#23455) (#23464) (3 weeks ago) <Brandon Morelli>
08:09:28 INFO:build_docs:e1e0c16fcb - Update monitoring-metricbeat.asciidoc (#21863) (#23233) (6 weeks ago) <DeDe Morton>
08:09:28 INFO:build_docs:fc48a3161e - Prepare 7.11 branch (#23164) (7 weeks ago) <Ivan Fernandez Calvo>
08:09:28 INFO:build_docs:c1154eb67a - Adding rate limit processor doc to processors list (#23096) (#23143) (7 weeks ago) <Shaunak Kashyap>
08:09:28 INFO:build_docs:0dae3d7abe - Allow node/namespace metadata to be disabled on kubernetes metagen and ensure add_kubernetes_metadata honors host (#23012) (#23127) (7 weeks ago) <Chris Mark>
08:09:28 INFO:build_docs:
08:09:28 INFO:build_docs:Recent commits in elasticsearch/Installation and Upgrade Guide:7.11:docs
08:09:28 INFO:build_docs:------------------------------------------------------------------------
08:09:28 INFO:build_docs:208e3c0ee60 - (7.11) [DOCS] Reuse timestamp reqs (#68299) (#68307) (68 minutes ago) <James Rodewig>
08:09:28 INFO:build_docs:81397561cc4 - [DOCS] Fix minor suggester doc errors (#68244) (#68304) (81 minutes ago) <James Rodewig>
08:09:28 INFO:build_docs:4b117fe473e - [DOCS] Remove outdated deprecated notes (#68246) (#68301) (82 minutes ago) <James Rodewig>
08:09:28 INFO:build_docs:07fd6eaf5a5 - [DOCS] Remove a ',' added to the end of the synonym in example docs (#68187) (#68285) (2 hours ago) <James Rodewig>
08:09:28 INFO:build_docs:cb1fc1d5501 - [DOCS] Fix typo (#68193) (#68289) (2 hours ago) <James Rodewig>
08:09:28 INFO:build_docs:a440fdf91ea - Update release notes for #67886 (#68224) (7 hours ago) <Martijn van Groningen>
08:09:28 INFO:build_docs:00721b9d0a2 - add possibility to mute yaml tests by operating system (#67681) (7 hours ago) <Hendrik Muhs>
08:09:28 INFO:build_docs:ab80d73ce4f - [DOCS] Fix SLM security privilege wording (#68238) (#68240) (3 days ago) <James Rodewig>
08:09:28 INFO:build_docs:72cc0d9e23a - [7.11] Add index creation version stats to cluster stats (#68141) (#68173) (4 days ago) <Lee Hinman>
08:09:28 INFO:build_docs:1ffcec1abdc - [DOCS] Fix typo (#68129) (#68164) (4 days ago) <James Rodewig>
08:09:28 INFO:build_docs:
08:09:28 INFO:build_docs:Recent commits in elasticsearch-hadoop/Installation and Upgrade Guide:7.11:docs/src/reference/asciidoc
08:09:28 INFO:build_docs:------------------------------------------------------------------------------------------------------
08:09:28 INFO:build_docs:b2a4ac62 - [DOCS] Remove 7.10.2 coming tag (#1573) (#1582) (3 weeks ago) <James Rodewig>
08:09:28 INFO:build_docs:95f28490 - [DOCS] Add 7.11.0 release notes (#1569) (3 weeks ago) <James Rodewig>
08:09:28 INFO:build_docs:261604fd - [DOCS] Add 7.10.2 release notes (#1568) (#1572) (3 weeks ago) <James Rodewig>
08:09:28 INFO:build_docs:be984452 - [DOCS] Fix links to Hadoop docs (#1564) (#1565) (8 weeks ago) <James Rodewig>
08:09:28 INFO:build_docs:2237fc29 - [DOCS] Add 7.10.1 release notes (#1560) (#1561) (8 weeks ago) <James Rodewig>
08:09:28 INFO:build_docs:7a215e9a - [DOCS] Add 7.10.0 release notes (#1553) (3 months ago) <James Rodewig>
08:09:28 INFO:build_docs:503e86a3 - [DOCS] Add 7.9.3 release notes (#1545) (#1547) (3 months ago) <James Rodewig>
08:09:28 INFO:build_docs:09346cde - [DOCS] Add 7.9.2 release notes (#1526) (#1533) (4 months ago) <James Rodewig>
08:09:28 INFO:build_docs:059e855e - Remove references to the field-types docs section. (#1522) (5 months ago) <Julie Tibshirani>
08:09:28 INFO:build_docs:d5aa98f3 - [DOCS] Add 7.9.1 release notes (#1518) (#1519) (5 months ago) <James Rodewig>
08:09:28 INFO:build_docs:
08:09:28 INFO:build_docs:Recent commits in kibana/Installation and Upgrade Guide:7.11:docs
08:09:28 INFO:build_docs:-----------------------------------------------------------------
08:09:28 INFO:build_docs:1ce672d2aac - [Vega Docs] Add experimental flag on the vega maps title (#89402) (#89595) (3 days ago) <Kaarina Tungseth>
08:09:28 INFO:build_docs:816c9f138df - [DOCS] Updates intro doc (#88376) (#89624) (4 days ago) <gchaps>
08:09:28 INFO:build_docs:4fea9eef7f3 - Update redirects.asciidoc (#84719) (#89575) (4 days ago) <Kaarina Tungseth>
08:09:28 INFO:build_docs:3b60d54fa60 - Remove change that didn't make 7.10 (#89548) (4 days ago) <Chris Roberson>
08:09:28 INFO:build_docs:1e658dd0bc7 - [maps] Update point to point docs with use cases (#89246) (#89496) (5 days ago) <Kent Marten>
08:09:28 INFO:build_docs:84a8c745432 - chore(NA): improve ts build refs performance on kbn bootstrap (#89333) (#89385) (5 days ago) <Tiago Costa>
08:09:28 INFO:build_docs:d801a7bb08e - JSON Body payload for the webhook connector in Alerts & Actions  (#89253) (#89375) (6 days ago) <Rashmi Kulkarni>
08:09:28 INFO:build_docs:5d60ad84d66 - [Maps] add video to maps docs (#89039) (#89310) (6 days ago) <Nathan Reese>
08:09:28 INFO:build_docs:7b4111e2bc6 - [docs] Resolving failed Kibana upgrade migrations (#80999) (#89295) (6 days ago) <Rudolf Meijering>
08:09:28 INFO:build_docs:87aca3a6f95 - Improve documentation for index pattern custom label (#89137) (#89262) (6 days ago) <Matthias Wilhelm>
08:09:28 INFO:build_docs:
08:09:28 INFO:build_docs:Recent commits in logstash/Installation and Upgrade Guide:7.11:docs
08:09:28 INFO:build_docs:-------------------------------------------------------------------
08:09:28 INFO:build_docs:036001b05 - Doc: Clarify minimum privileges for logstash_writer role (#12613) (#12624) (3 days ago) <Karen Metts>
08:09:28 INFO:build_docs:7adb93882 - [forwardport 7.11] Release notes for 7.10.2 (#12553) (3 weeks ago) <Joao Duarte>
08:09:28 INFO:build_docs:8746cdeda - plugin: adds `:validate => :field_reference` (#12459) (#12521) (7 weeks ago) <Ry Biesemeyer>
08:09:28 INFO:build_docs:8550059a2 - Forward port of 7.10.1 release notes to 7.x (8 weeks ago) <andsel>
08:09:28 INFO:build_docs:f50cc38ea - [Doc] expanded the description of how to obtain the logger's names form Java and Ruby class names (10 weeks ago) <andsel>
08:09:28 INFO:build_docs:cf7e2e308 - [Forward] Doc:Release notes 7.10.0 to 7.x (#12433) (3 months ago) <Karen Metts>
08:09:28 INFO:build_docs:5037fd23e - [Backport] Doc: Add bundled JDK info (#12417) (3 months ago) <Karen Metts>
08:09:28 INFO:build_docs:d473e0b7c - [backport 7x] Doc: wildcards in xpack.management.pipeline.id (#12416) (3 months ago) <kaisecheng>
08:09:28 INFO:build_docs:54505a9b4 - [Backport]Doc:Restructure troubleshooting docs and add plugin tracing (#12405) (3 months ago) <Karen Metts>
08:09:28 INFO:build_docs:50428785c - [Backport][Doc]Relocate logstash-to-cloud info for increased visibility (#12400) (3 months ago) <Karen Metts>
08:09:28 INFO:build_docs:
08:09:28 INFO:build_docs:Used /doc/elastic+observability-docs+pull-request+build-docs for observability-docs/Installation and Upgrade Guide:7.11:docs/en/observability
08:09:28 INFO:build_docs:Recent commits in docs/Installation and Upgrade Guide:HEAD:shared/versions/stack/{version}.asciidoc
08:09:28 INFO:build_docs:---------------------------------------------------------------------------------------------------
08:09:28 INFO:build_docs:
08:09:28 INFO:build_docs:
08:09:28 INFO:build_docs:Recent commits in docs/Installation and Upgrade Guide:HEAD:shared/attributes.asciidoc
08:09:28 INFO:build_docs:-------------------------------------------------------------------------------------
08:09:28 INFO:build_docs:3f7f7478c72c - (HEAD -> master, origin/master, origin/HEAD) Add ecs-logging-go-logrus to the build (#2047) (5 days ago) <Brandon Morelli>
08:09:28 INFO:build_docs:b0bf7f15315a - Add ecs-logging-go-zap to the build (#2038) (2 weeks ago) <Brandon Morelli>
08:09:28 INFO:build_docs:ff54fd46a703 - Add ecs-logging-python to the build (#2037) (3 weeks ago) <Brandon Morelli>
08:09:28 INFO:build_docs:405a0611ac84 - [DOCS] Add attribute for Fleet guide (#2028) (7 weeks ago) <James Rodewig>
08:09:28 INFO:build_docs:d6a4ee0d1c43 - Add ECS .NET & Ruby documentation (#2018) (7 weeks ago) <Brandon Morelli>
08:09:28 INFO:build_docs:b2684f9afd67 - docs: fix ecs-logging link (#2021) (8 weeks ago) <Brandon Morelli>
08:09:28 INFO:build_docs:0129581ccda5 - docs: add ecs-logging (#2017) (8 weeks ago) <Brandon Morelli>
08:09:28 INFO:build_docs:0601a99e368e - Add ECS Logging Java Reference to doc build (#2010) (2 months ago) <Brandon Morelli>
08:09:28 INFO:build_docs:b637c239a20c - Add attributes for searchable snapshots (#1979) (3 months ago) <debadair>
08:09:28 INFO:build_docs:7e92c2ffd50e - Add new Observability attributes (#1980) (4 months ago) <Brandon Morelli>
08:09:28 INFO:build_docs:
08:09:28 INFO:build_docs:
08:09:28 INFO:build_docs:asciidoctor: WARNING: invalid reference: enhanced-host-details
08:09:28 INFO:build_docs:
08:09:28 INFO:build_docs:Child exited with 255 at lib/ES/Util.pm line 666.
08:09:42 runbld>>> <<<<<<<<<<<< SCRIPT EXECUTION END <<<<<<<<<<<<
```
</details>

L1 is the hint: `08:09:28 INFO:build_docs:ERROR building Installation and Upgrade Guide version 7.11`. All internal links have to be external links because this content is reused in the I&UG.

**Comment by tbragin** [comment]
LGTM

**Comment by EamonnTP** [comment]
Hi @mukeshelastic Could you provide the URL for the runtime fields blog?

**Comment by bmorelli25** [comment]
> Could you provide the URL for the runtime fields blog?

@EamonnTP, this looks to be the runtime fields blog URL: elastic.co/blog/introducing-elasticsearch-runtime-fields

**Review by bmorelli25 — APPROVED** [review]

File: docs/en/observability/whats-new.asciidoc [hunk 5]
```diff
@@ -3,19 +3,248 @@
...
+In {minor-version}, the new service overview page is now your default starting point
+for investigating any services monitored by Elastic APM agents or with OpenTelemetry and Jaeger.
+
+[role="screenshot"]
+image::images/apm-service-overview.png[Service overview page]
+
+The improved APM service overview page provides you with: 
+
+* High-level details including service, runtime and framework versions, container
+and Kubernetes information, along with cloud metadata, if applicable
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:39]:
```suggestion
and Kubernetes information, along with cloud metadata, if applicable.
```

File: docs/en/observability/whats-new.asciidoc [hunk 6]
```diff
@@ -3,19 +3,248 @@
...
+* Latency timeline overlapped with deployment events, alert violations, and anomalies
+which occurred during the same period.
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:41]:
```suggestion
that occurred during the same period.
```

File: docs/en/observability/whats-new.asciidoc [hunk 7]
```diff
@@ -3,19 +3,248 @@
...
+
+In {minor-version}, we’re delighted to introduce an enhanced view of hosts or VMs to the
+Metrics app. Without leaving the Inventory page, you can view the following information about the host or VM:
+
+* Time charts of key host metrics: CPU, memory, load, and network
+* Logs generated by the host or the services running on the host
+* Top processes running on the host by CPU or memory
+* Host metadata: host, cloud, and agent information
+
+The enhanced host details help accelerate root cause analysis by tightly aligning how the
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:88]: would be better to break this into two sentences. Otherwise, I think the subj/verb agreement is off.

It's a little hard to parse the two clauses.

File: docs/en/observability/whats-new.asciidoc [hunk 8]
```diff
@@ -3,19 +3,248 @@
...
+Your application custom metrics are now monitored with zero
+effort whenever you monitor your Python applications with APM. Once you hook up custom
+metrics, you can quickly build a {kib} dashboard using TSVB or Lens to analyze them
+and correlate with other performance metrics.
+
+[discrete]
+== {log-driver-long} (GA)
+
+In {minor-version}, we are excited to announce that the https://hub.docker.com/plugins/elastic-logging-plugin[Docker logging plugin]
+for enabling simpler UX for application logging is now generally available.
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:180]: will all readers know what we mean by UX? Might be better to write it out just in case.

File: docs/en/observability/whats-new.asciidoc [hunk 9]
```diff
@@ -3,19 +3,248 @@
...
+
+You can use the {log-driver-long} to forward logs to {es}, {ls}, Kafka, or Redis,
+for all Docker containers or on a per-container basis. Unlike other Beats, the {log-driver-long} requires no
+elevated permissions to read container logs, and the installation is performed entirely
+within the Docker CLI.
+
+[discrete]
+== {filebeat} and {metricbeat} modules (GA)
+
+For {minor-version} we’re delighted to announce the general availability of the following modules:
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:190]: Minor style comment, but I think we say "delighted" a little too frequently.

File: docs/en/observability/whats-new.asciidoc [hunk 10]
```diff
@@ -3,19 +3,248 @@
...
+* {filebeat-ref}/filebeat-module-microsoft.html#_m365_defender_fileset_settings[{filebeat} Microsoft 365 Defender]
+* {filebeat-ref}/filebeat-module-microsoft.html#_defender_atp_fileset_settings[{filebeat} Defender API]
+* {filebeat-ref}/filebeat-module-google_workspace.html[{filebeat} Google Cloud Workspace]
+
+These integrations are now stable, production-ready, and fully supported!
+
+[discrete]
+== Natively collect AWS Fargate metrics
+
+If you’re running Amazon’s ECS or EKS the chances are you’re using AWS Fargate to manage these
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:204]:
```suggestion
If you’re running Amazon’s ECS or EKS, the chances are you’re using AWS Fargate to manage these
```

File: docs/en/observability/whats-new.asciidoc [hunk 11]
```diff
@@ -3,19 +3,248 @@
...
+deployments, because Fargate removes the responsibility of provisioning and managing the
+underlying EC2 infrastructure. You only need to specify your containers and tasks.
+
+In {minor-version}, we’ve added a metric collection from AWS Fargate service to our cloud integrations list.
+The new {metricbeat-ref}/metricbeat-module-awsfargate.html[AWS `fargate`] {metricbeat} module collects
+container metrics and metadata from the Fargate’s task metadata endpoints and allows you to
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:210]: Fargate is a trademark and therefore should not be possessive.

**Review by dedemorton — COMMENTED** [review]
I have some really minor comments, but overall this looks great! Thanks for pulling this together.

**Comment by EamonnTP** [comment]
Thanks @dedemorton for your comments!

**Review by dedemorton — APPROVED** [review]
LGTM!

**Comment by tbragin** [comment]
@EamonnTP I just found out that one feature did not fully make it due to a bug we found. It's the one about synchronizing time across apps. Can we strike that section from what's new? We'll re-add it in 7.12, when it's ready.

**Comment by EamonnTP** [comment]
@tbragin No problem, I'll remove it now.
